index.js

```diff
@@ -1,29 +1,74 @@
-/*NOTE: the size of the files extracted from the <input type="file"> element
-            is measured in Bytes. Thus, all convertions from Bytes, to KiloBytes, MegaBytes 
-            and so on must take that into account. */
-
 const KB = 1024;
 const MB = KB*KB;
 const GB = KB*MB;
 
+const supported_extensions = ["jpg",
+    "jpeg","png","gif","bmp","tiff",
+    "svg","webp","ico","heif","avif",
+    "JPEG","PNG","GIF","BMP","TIFF",
+    "SVG","WEBP","ICO","HEIF","AVIF",
+    "JPG"];
+
 const AVAIL_SPACE = "available_space";
 const size_suffix = "&&fileSize";
 
-const total_space = 10*MB;/*measured in Byte units.*/
+const total_space = 10*MB;
+let available_space;
+
+const size_to_string = size_bytes => {
+    if(size_bytes/GB>1){
+        return `${(size_bytes/GB).toFixed(1)} GB`;
+    }
+    if(size_bytes/MB>1){
+        return `${(size_bytes/MB).toFixed(1)} MB`;
+    }
+    if(size_bytes/KB>1){
+        return `${(size_bytes/KB).toFixed(1)} KB`;
+    }
+    return `${size_bytes} Bytes`;
+}
+
+const sizeKey = fileName => `${fileName}${size_suffix}`;
+
+const setAvailableSpace = num => localStorage.setItem(AVAIL_SPACE,num);
+
+const setBarWidth = () => {
+    let percentage = 100*(total_space-available_space)/total_space;
+    document.getElementsByClassName("gradient-bar")[0].setAttribute("style",`width:${percentage}%`);
```

Comment:
Keep your naming consistents : 

`let percentage = 100*(totalSpaceAvailableSpace)/totalSpace;`

Also, `total-space` sounds like a constant? if so change to `TOTAL_SPACE`

You could also change the name of the variable into `total_space-available` and it would be just as understandable (or, `totalSpaceAvailable as mentioned above)